apps/platform/src/components/AssociationsToolkit/components/HeaderControls/HeaderControls.tsx

```diff
@@ -53,20 +88,25 @@ function HeaderControls({ cols = [] }) {
               <Typography variant="subtitle2">Weight</Typography>
             </Box>
             <Box>
-              <Typography variant="subtitle2">Require</Typography>
+              <Typography variant="subtitle2">
+                <Tooltip
+                  title="Enabling this will return either of the selected data sources."
+                  showHelpIcon
+                  style={null}
+                >
```

Comment:
Where this change comes from ?

Reply:
Issue: https://github.com/opentargets/issues/issues/3422